### ISSUE TITLE
.inclusivelintignore is not pointing to the correct path in GitHub

### ISSUE THREAD

**Issue by squassina** [inclusivelint/inclusivelint#28]
The create_ignore_words_list is pointing to the wrong place. Instead of:
```bash
function create_ignore_words_list() {
    while read line; do
        IGNORE_DICTIONARY[$line]=1
    done < <( cat ~/.inclusivelint/.inclusivelintignore )
}
```
it should be:
```bash
function create_ignore_words_list() {
    while read line; do
        IGNORE_DICTIONARY[$line]=1
    done < <( cat ${BASE_PATH}/.inclusivelint/.inclusivelintignore )
}
```
So it would work in GitHub as well.

**Comment by squassina** [comment]
Assigned this one to myself.

**Comment by squassina** [comment]
fixed in PR #29
